samples/benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 3.13.1)
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(benchmark)
+
+target_sources(app PRIVATE src/main.c)
+add_subdirectory_ifdef(CONFIG_DHRYSTONE dhrystone)
```

Comment:
samples/benchmark is a new directory, right?

I believe there is a pre-existing benchmark directory somewhere, dhyrstone would be more appropriate there.

Reply:
@SebastianBoe Do you mean ./tests/benchmarks?

Reply:
yes

Reply:
OK, will do.
In addition, the dhrystone failed to build with zephyr-sdk-0.10.0-rc2.
BTW, the same code could be built successfully with zephyr-sdk-0.9.5.

[ 95%] Linking C executable zephyr_prebuilt.elf
Memory region Used Size Region Size %age Used
FLASH: 34176 B 1 MB 3.26%
SRAM: 15640 B 192 KB 7.95%
IDT_LIST: 120 B 2 KB 5.86%
/opt/zephyr-sdk/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/8.2.0/../../../../arm-zephyr-eabi/bin/ld: /opt/zephyr-sdk/arm-zephyr-eabi/arm-zephyr-eabi/lib/thumb/v7e-m/nofp/libc.a(lib_a-gettimeofdayr.o): in function _gettimeofday_r': /home/buildslave/src/github.com/zephyrproject-rtos/sdk-ng/build/build_arm/.build/arm-zephyr-eabi/src/newlib/newlib/libc/reent/gettimeofdayr.c:63: undefined reference to _gettimeofday'
collect2: error: ld returned 1 exit status
make[2]: *** [zephyr/zephyr_prebuilt.elf] Error 1
make[1]: *** [zephyr/CMakeFiles/zephyr_prebuilt.dir/all] Error 2
make: *** [all] Error 2

Steps to reproduce the behavior:

    apply #13383
    mkdir build; cd build
    cmake -DBOARD=stm32f4_disco ../samples/benchmark
    make

Environment (please complete the following information):

    OS: Ubuntu 14.04.5 LTS
    Toolchain: zephyr-sdk-0.10.0-rc2
    Commit SHA: 7077602

Any help would be greatly appreciated.

Reply:
@SebastianBoe 
The build error is gone with new zephyr-sdk-0.10.0.
Thanks for your review!